config/ad_config.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/python3
+# coding=utf-8
+"""
+Automate demultiplex configuration. Contains the following settings:
+- General settings used across modules
+- Demultiplexing script-specific settings
+- Setoff workflows script-specific settings
+"""
+import os
+import datetime
+from pygit2 import Repository
```

Comment:
I like this use of pygit2 for detecting if production or development branches are being used